utils/constants.js

```diff
@@ -5,10 +5,19 @@ export const BIRTH_DATE_FIELD_CODE = 'NI0QRzJvQ0k';
 // Programs
 export const NEARKONTAKT_PROGRAM_CODE = 'DM9n1bUw8W8';
```

Comment:
_CODE is a bit misleading when it comes to metadata objects, as this is the "ID" or "UID" we are hard coding in these constants. The "Code" is a separate property on most metadata objects that either contains nothing, or contains a user defined unique code.

Reply:
Changed to ID on all instead. CODE was chosen without knowing the specific meaning of it in dhis2